TimeFormat.java

```diff
@@ -9,9 +9,26 @@ public static void main(String[] args) {
 		// It concatenates the empty string "" with the leftmost hour-digit. 
 		// It then concatenates the resulting string with the rightmost hour-digit,
 		// and then uses parseInt to cast the resulting string as an int.
-		int hours = Integer.parseInt("" + args[0].charAt(0) + args[0].charAt(1));
-		// Does the same with the minutes part of the input.
-		int minutes = Integer.parseInt("" + args[0].charAt(3) + args[0].charAt(4));
-        // Replace this comment with the rest of your code
+			int hours = Integer.parseInt("" + args[0].charAt(0) + args[0].charAt(1));
+			// Does the same with the minutes part of the input.
+			int minutes = Integer.parseInt("" + args[0].charAt(3) + args[0].charAt(4));
+			// Minutes formating 
+			String finalMinutesFormat = "" + minutes;
+			if (minutes < 10) {
+				finalMinutesFormat = "0" + minutes;			
+			}
+			// PM calc
+			if (hours > 12) {

```

Comment:
Use as we said in Recitation and WS. there is a difference between 
3 ifs and if, else if, else. 
```
if (hours > 12) {
	System.out.println((hours - 12) +  ":" + finalMinutesFormat + " PM");
} else if (hours < 12) { 
	System.out.println(hours + ":" + finalMinutesFormat + " AM");
} else  {
	System.out.println(hours + ":" + finalMinutesFormat + " PM");
}
```